shell/platform/darwin/macos/framework/Headers/FlutterPluginRegistrarMacOS.h

```diff
@@ -36,11 +36,9 @@ FLUTTER_DARWIN_EXPORT
 @property(nonnull, readonly) id<FlutterTextureRegistry> textures;
 
 /**
- * The default view displaying Flutter content.
+ * The view displaying Flutter content.
```

Comment:
Given this is a user-facing header file, I'd add more details. We should make it clear this makes single view assumptions. Maybe consider something like:

> The implicit view that displays Flutter content.
>
> This property is provided for backwards compatibility for apps
> that assume a single view.
>
> This method may return |nil|, for instance in a headless environment
> or for apps that do not assume a single view.

Reply:
The comment on `FlutterView.h` is excellent and may be worth copying here as well:

```
 * Some single-view APIs will eventually be replaced by their multi-view
 * variant. During the deprecation period, the single-view APIs will coexist with
 * and work with the multi-view APIs as if the other views don't exist.  For
 * backward compatibility, single-view APIs will always operate on the view with
 * this ID. Also, the first view assigned to the engine will also have this ID.
```

Reply:
Done. I was thinking to remove all references to multi-view since from the view of the developers, multi-view isn't supported at all. But I think announcing the plan beforehand also works and makes developers prepared too.